homeassistant/components/plex/media_player.py

```diff
@@ -35,26 +35,40 @@
 from .const import (
     CONF_USE_EPISODE_ART,
     CONF_SHOW_ALL_CONTROLS,
+    CONF_SERVER_IDENTIFIER,
     DOMAIN as PLEX_DOMAIN,
     NAME_FORMAT,
     PLEX_MEDIA_PLAYER_OPTIONS,
     SERVERS,
 )
 
-SERVER_SETUP = "server_setup"
-
-_CONFIGURING = {}
 _LOGGER = logging.getLogger(__name__)
 
 
-def setup_platform(hass, config, add_entities_callback, discovery_info=None):
-    """Set up the Plex platform."""
-    if discovery_info is None:
-        return
+def async_setup_platform(hass, config, add_entities_callback, discovery_info=None):
```

Comment:
Use `async def`.

Please rename `add_entities_callback` to `async_add_entities`.